code-pushup.preset.ts

```diff
@@ -156,14 +157,25 @@ export const jsDocsCoreConfig = (
   ),
 });
 
+export async function eslintConfigFromPublishableNxProjects(): Promise<
+  ESLintTarget[]
+> {
+  const { createProjectGraphAsync } = await import('@nx/devkit');
+  const projectGraph = await createProjectGraphAsync({ exitOnError: false });
+  return nxProjectsToConfig(
+    projectGraph,
+    project => project.tags?.includes('publishable') ?? false,
+  );
+}
+
 export const eslintCoreConfigNx = async (
   projectName?: string,
 ): Promise<CoreConfig> => ({
   plugins: [
     await eslintPlugin(
       await (projectName
         ? eslintConfigFromNxProject(projectName)
-        : eslintConfigFromAllNxProjects()),
+        : eslintConfigFromPublishableNxProjects()),
```

Comment:
What's wrong with linting non-publishable projects? :thinking:

Reply:
I would only include projects that are also tracked in the monorepo setup to have a good comparison base.

Reply:
Agreed, the same projects should be included in both standalone and monorepo modes. Question is whether to include every project in both, or only published packages in both. 

For simplicity, I'm leaning towards collecting all projects. It also gives a little bit more scale to play with - caching impact will be more noticable when more projects are involved.

Reply:
What about tagging all projects with quality levels:
- their-a - released code
- their-b - released code secondary
- their-c - tooling, testing, pers, code-pushup, 
- their-d - examples

Example for a `project.json`:
```json
{
    tags: [ "checks:their-a" ] 
}
```

Reply:
One more thing to consider, as this PR is concerned with ESLint. The rules configured from `@code-pushup/eslint-config` include many adjustments for "test code", so we already distinguish prod and non-prod with regards to how strict our rule set is.